style: design homepage wireframe

pallavithakare28 opened the pull request:
# Description
I have introduced two wireframe designs for the homepage namely, design-1 & design-2

Fixes #1034 

# Type of Change:
- User Interface

# Mocks
`Design-1`
![design-1](https://user-images.githubusercontent.com/62203552/92268631-5d258580-ef00-11ea-82b7-91c9d5dd6909.png)

`Design-2`
![design-2](https://user-images.githubusercontent.com/62203552/92268645-66165700-ef00-11ea-8ad5-582067585d7d.png)

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

pallavithakare28 commented:
@Kajol-Kumari I have created this new PR, and Can you please review it.?

Kajol-Kumari commented:
Hey @pallavithakare28 please do the following:
1. use the updated logo with don't contain the word `Systers` in it. You can get it from [here](https://github.com/anitab-org/vms/pull/1000)
2. you design needs to clean in terms of proper spacing and don't add `news` , `contacts` on the homepage instead we can keep a slider on the home page for showing `achievements`

Kajol-Kumari commented:
@anitab-org/design-team can you please review this PR

pallavithakare28 commented:
@Kajol-Kumari @anitab-org/design-team PR page is showing error 404. Can you please help me

Kajol-Kumari commented:
> @Kajol-Kumari @anitab-org/design-team PR page is showing error 404. Can you please help me

@pallavithakare28 sorry but when are you getting 404?

pallavithakare28 commented:
> @anitab-org/design-team can you please review this PR

Sorry @Kajol-Kumari buti for this PR I am getting a 404

Kajol-Kumari commented:
> > @anitab-org/design-team can you please review this PR
> 
> Sorry @Kajol-Kumari buti for this PR I am getting a 404

Ah, @pallavithakare28 that is not the PR link, that's the way to tag vms's design team to review this PR.

pallavithakare28 commented:
@Kajol-Kumari  ok mam and can you please the description of this once? Because I have uploaded a new wireframe design.

keshakaneria reviewed (dismissed):
@pallavithakare28 @Kajol-Kumari  Looks good 👍

Kajol-Kumari commented:
@pallavithakare28 please keep the design you have shown in the PR description and remove the other one. Also please squash the commits in a single one.

pallavithakare28 commented:
@Kajol-Kumari please review the PR as I have removed other designs and also squashed the commits.  And also I would like to say thank you for your guidance. Thank you, mam.

pallavithakare28 commented:
> @pallavithakare28 @Kajol-Kumari Looks good

@keshakaneria Thank you mam, Can please review this PR.?

keshakaneria reviewed (approved):
Squashed! Well done! It's ready to get the final review by maintainers :) 👍

Kajol-Kumari reviewed (approved):
LGTM :+1:

pallavithakare28 commented:
Hey @Kajol-Kumari minimum how many approval are required to merge this PR? Just asking for a knowledge :)

keshakaneria commented:
> Hey @Kajol-Kumari minimum how many approval are required to merge this PR? Just asking for a knowledge :)

It requires minimum 2 reviews. We can change the label to ready to merge i suppose

SanketDG reviewed (commented):
I would again not honestly merge this, since this is honestly very ephemeral by nature.

This is great work, don't get me wrong, but this PNG file can't be edited on, hence make no sense for the wireframe to be inside the repo.

I'd very much prefer that I have the designs are discussed in the issues and then we implement the design.

pallavithakare28 commented:
@SanketDG Ok that means there's no meaning of this PR?

SanketDG commented:
> Ok that means there's no meaning of this PR?

I should not be the only person taking this decision, this is just my opinion, please feel free to ask on Zulip about this.

I also don't know who to tag for VMS, apologies

Kajol-Kumari commented:
Hey @pallavithakare28 so won't be able to merge this PR but we will consider it while evaluating. So don't worry, your hard work will not go in vain :) 

\cc: @satya7289

pallavithakare28 commented:
> Hey @pallavithakare28 so we won't be able to merge this PR but we will consider it while evaluating. So don't worry, your hard work will not go in vain :)
> 
> \cc: @satya7289

Thank you @Kajol-Kumari :)

alpha2320 commented:
@SanketDG  can i redesign this in figma and adobe xd and then can you merge it?
